Tribler/Test/Core/base_test.py

```diff
@@ -1,5 +1,12 @@
 from Tribler.Test.test_as_server import AbstractServer
 
 
+class MockObject(object):
```

Comment:
Yeah I agree, I got a bit tired of defining separate classes every time and last week, @whirm told me about creating classes that derive from `object` which is an easy way to mock objects so I decided to go with this approach.